### PR TITLE
Use hash_equals to compare hashes

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

File: source/user_manual.rst [hunk 1]
```diff
@@ -1005,7 +1005,7 @@ And here's a sample in PHP:
         }
 
         // returns true if signature is valid
-        return hash_hmac('sha256', $timestamp . $token, $signingKey) === $signature;
+        return \hash_equals(hash_hmac('sha256', $timestamp . $token, $signingKey), $signature);
```
Comment (alystair) [inline source/user_manual.rst]: why a leading slash?

Comment (Nyholm) [reply source/user_manual.rst]: It is to make sure we use the function from the global namespace.

File: source/user_manual.rst [hunk 2]
```diff
@@ -1000,12 +1000,12 @@ And here's a sample in PHP:
...
     {
         // check if the timestamp is fresh
-        if (abs(time() - $timestamp) > 15) {
+        if (\abs(\time() - $timestamp) > 15) {
             return false;
         }
 
         // returns true if signature is valid
-        return \hash_equals(hash_hmac('sha256', $timestamp . $token, $signingKey), $signature);
+        return \hash_equals\(hash_hmac('sha256', $timestamp . $token, $signingKey), $signature);
```
Comment (alystair) [inline source/user_manual.rst]: Did you mean` \hash_equals(\hash_hmac( ...` ?

Comment (Nyholm) [reply source/user_manual.rst]: Yes. Sorry. 

Editing this large file on the phone is not easy =)

**Comment by Nyholm** [comment]
PR is updated and perfect. 

Thank you for the reivew.

**Pull request merged**